oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/otlp/OpenTelemetryMetricRequestProcessor.java

```diff
@@ -161,6 +163,24 @@ private static Map<Double, Long> buildBuckets(
         return result;
     }
 
+    private static Map<Double, Long> buildBucketsFromExponentialHistogram(
```

Comment:
Please add detailed and well formatted comments about the converting logic. It is new.